install.py

```diff
@@ -204,7 +204,10 @@ def go(ui, args, answerfile_address, answerfile_script):
         results['extra-repos'] += extra_repo_defs
         logger.log("Driver repos: %s" % str(results['extra-repos']))
 
+        mounter = disktools.DeviceMounter()
```

Comment:
Implementing the context manager protocol would make this nicer.

Reply:
done